environment.py

```diff
@@ -213,7 +213,7 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_DSS_WORKERS': '8',
 
         # Whether to create a subscription to DSS during deployment. Set this
-        # variable to 1 to enable `make deploy` to subscribe the indexer in the
+        # variable to 1 to enable `make package` to subscribe the indexer in the
```

Comment:
Same as above: this statement is actually not even correct, since make package doesn't actually deploy the lambda. Do you think you could fix it?